terraform/test/fixtures/oidc_connector_2_update_only_redirect_urls.tf

```diff
@@ -0,0 +1,21 @@
+resource "teleport_oidc_connector" "test_multiple_redirects" {
+    metadata = {
+        name    = "test_multiple_redirects"
+        expires = "2022-10-12T07:20:50Z"
+        labels  = {
+            example = "yes"
+        }
+    }
+
+    spec = {
+        client_id = "client"
+        client_secret = "value"
+    
+        claims_to_roles = [{
+            claim = "test"
+            roles = ["terraform"]
+        }]
+
+        redirect_urls = [ "https://example.com/redirect", "https://example.com/redirect2" ]
```

Comment:
Since `redirect_url`/`redirect_urls` is now required by the Teleport API, we should also add it to `oidc_connector.tf.example`.

P.S. I pushed that change before our recent conversation about API compatibility, sorry...

Reply:
Nice catch, thank you